phpBB/includes/acp/acp_database.php

```diff
@@ -98,50 +101,105 @@ function main($id, $mode)
 
 						$filename = 'backup_' . $time . '_' . unique_id();
 
-						$extractor = $phpbb_container->get('dbal.extractor');
-						$extractor->init_extractor($format, $filename, $time, $download, $store);
+						try
+						{
+							$extractor = $phpbb_container->get('dbal.extractor');
+							$extractor->init_extractor($format, $filename, $time, $download, $store);
 
-						$extractor->write_start($table_prefix);
+							$extractor->write_start($table_prefix);
 
-						foreach ($table as $table_name)
-						{
-							// Get the table structure
-							if ($structure)
+							foreach ($table as $table_name)
 							{
-								$extractor->write_table($table_name);
+								// Get the table structure
+								if ($structure)
+								{
+									$extractor->write_table($table_name);
+								}
+								else
+								{
+									// We might wanna empty out all that junk :D
```

Comment:
Please rephrase that comment. I'll be happy once all of those weird comments are gone ;)